Tests/SwiftCurrent_SwiftUITests/PersistenceTests.swift

```diff
@@ -0,0 +1,513 @@
+//
+//  PersistenceTests.swift
+//  SwiftCurrent_SwiftUI
+//
+//  Created by Tyler Thompson on 7/12/21.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//
+
+import XCTest
+import SwiftUI
+import ViewInspector
+
+import SwiftCurrent
+@testable import SwiftCurrent_SwiftUI // testable sadly needed for inspection.inspect to work
+
+@available(iOS 14.0, macOS 11, tvOS 14.0, watchOS 7.0, *)
+final class PersistenceTests: XCTestCase {
+    // MARK: RemovedAfterProceedingTests
+    func testRemovedAfterProceeding_OnFirstItemInAWorkflow() throws {
+        struct FR1: View, FlowRepresentable, Inspectable {
+            var _workflowPointer: AnyFlowRepresentable?
```

Comment:
Because of where we are at at this point, I wonder if it is safe to change these to weak to prevent the retention cycle or if that could throw our tests into an unknown state?
```suggestion
            weak var _workflowPointer: AnyFlowRepresentable?
```

Reply:
This isn't really a problem in our tests at this time, so I think we'll leave it for now.